### DISCUSSION TITLE
RMSD on alpha-carbon

### DISCUSSION THREAD

**Discussion by yazdan12555** [charnley/rmsd#108] [Q&A] [answered]
Hi. 
Does the code have the ability to run RMSD only on alpha carbons?
i couldn't find a related option by running "calculate_rmsd --help"

**Comment by charnley** [comment]
The feature is not available using the command line interface. However, 

A) you can read the PDB with Python and use the functions from the library

B) Remove all other atoms from the PDB file and use the command line interface

If it is a feature you need, you are welcome to open an issue.

**Reply by yazdan12555** [reply]
Hi Jimmy,

Thanks so very much for the quick response. I got my answer from your suggestion :)

Best,

Yazdan

**Comment by charnley** [comment, marked as answer]
You can now call calculate_rmsd with ``--only-alpha-carbons``

    calculate_rmsd --only-alpha-carbons file1.pdb file2.pdb

and if the the atoms are not in the right corner, you can call it with ``--reorder``

    calculate_rmsd --reorder --reorder-method qml --only-alpha-carbons file1.pdb file2.pdb

**Reply by yazdan12555** [reply]
Excellent! Thanks so very much.
